sql/migrations/20241121142159_v0.52.0.sql

```diff
@@ -10,6 +10,9 @@ CREATE TYPE "WorkflowTriggerCronRefMethods" AS ENUM ('DEFAULT', 'API');
 CREATE TYPE "WorkflowTriggerScheduledRefMethods" AS ENUM ('DEFAULT', 'API');
 -- Modify "WorkflowTriggerCronRef" table
 ALTER TABLE "WorkflowTriggerCronRef" ADD COLUMN "name" text NULL, ADD COLUMN "id" uuid NOT NULL, ADD COLUMN "method" "WorkflowTriggerCronRefMethods" NOT NULL DEFAULT 'DEFAULT', ADD CONSTRAINT "WorkflowTriggerCronRef_parentId_cron_name_key" UNIQUE ("parentId", "cron", "name");
+
+UPDATE "WorkflowTriggerCronRef" SET "name" = '' WHERE "name" IS NULL;
```

Comment:
fixing a broken past migration?

Reply:
since we haven't released this as latest, this is alright -- this just makes the migration more zero-downtime